### PR TITLE
Add @slash_command.autocomplete() example.

### PR DESCRIPTION
**Pull request by Victorsitou**
## Summary
This code isn't tested but I think it works.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by abhigyantrips** [comment]
I guess https://github.com/EQUENOS/disnake/blob/master/tests/cogs/slash_commands.py#L17-L30 already exists.

**Comment by Victorsitou** [comment]
> I guess https://github.com/EQUENOS/disnake/blob/master/tests/cogs/slash_commands.py#L17-L30 already exists.

I know, but I think people look into examples folder instead of tests, maybe I'm wrong?

File: examples/slash_commands/param.py [hunk 1]
```diff
@@ -165,6 +165,20 @@ async def autocomplete(
...
+
+
+@bot.slash_command()
+async def languages(inter: disnake.ApplicationCommandInteraction, language: str):
+    ...
+
+
+@languages.autocomplete("language")
+async def language_autocomp(inter: disnake.ApplicationCommandInteraction, string: str):
+    return ["Python", "JavaScript", "TypeScript", "Java", "Rust", "Lisp", "Elixir"]
```
Comment (EQUENOS) [inline examples/slash_commands/param.py:179]: The whole point of autocomplete here isn't really shown, I think you can do something like this:
```py
return [lang for lang in langs if string in lang.lower()]
```

Comment (Victorsitou) [reply examples/slash_commands/param.py:179]: My bad, sorry. 
Are type hints necessary here?

Comment (EQUENOS) [reply examples/slash_commands/param.py:179]: Type hints are nice and helpful

Comment (thesadru) [reply examples/slash_commands/param.py:179]: But please in moderation. Looking at you LEv145.

**Comment by thesadru** [comment]
It was just a problem with formatting combined with some misdirection caused by LEv145's antics.

**Comment by Victorsitou** [comment]
> It was just a problem with formatting combined with some misdirection caused by LEv145's antics.

Thank you Sadru

**Pull request merged**